templates/web/src/sdk.ts.twig

```diff
@@ -9,6 +9,55 @@ type Headers = {
     [key: string]: string;
 }
 
+type RealtimeResponse = {
+    type: "error"|"event"|"connected"|"response";
+    data: RealtimeResponseAuthenticated|RealtimeResponseConnected|RealtimeResponseError|RealtimeResponseEvent;
+}
+
+type RealtimeRequest = {
+    type: "authentication";
+    data: RealtimeRequestAuthenticate;
+}
+
+type RealtimeResponseEvent = {
+    event: string;
+    channels: string[];
+    timestamp: number;
+    payload: unknown;
```

Comment:
Would it be possible to use generic types? Usage would be something like:

```typescript
type PostsCollection = { title: string; text: string; likes: number };

sdk.subscribe<PostsCollection>("files", payload => {
  console.log(payload.tilte); // ERROR: Type of PostsCollection does not have 'tite' property. Did you mean 'title'?
});
```

Reply:
Sadly not, since you are able to subscribe to multiple channels👍🏻 

```typescript
sdk.subscribe<PostsCollection>(["files", "account"], payload => {
  console.log(payload.tilte); // ERROR: Type of PostsCollection does not have 'tite' property. Did you mean 'title'?
});
```

Reply:
Anyway, I think we could still add it. Will be in the users responsibilty to use it correctly.

Reply:
I see. I still think it is important.. Hmm 🤔 Can we have 2 subscribe methods? One for `string`, one for `string[]`.
This way we would type only the `string` one with a generic type, so people can use it if they want. I would personally like to split events into 10 subscriptions than be without types for my collections.

Another idea, what about this one?

```typescript
type PostsCollection = { title: string; text: string; likes: number };
type LikesCollection = { title: string; text: string; likes: number };

sdk.subscribe<PostsCollection | LikesCollection>(["collection:posts", "collection:likes"], (payload) => {
  // stuff
});
```

If you need multiple subscriptions inside one subscribe method, put in both types. Then it is up to you to know which one you have.. But the good solution is to make one `.subscribe` for one type.. The array solution seems good only if you have same or similar models for multiple collections.

Reply:
Added generic type, if they add multiple channels with different types - its the devs task to handle it for now 👍🏻